tests/unit-tests/test_gnn.py

```diff
@@ -1059,8 +1059,10 @@ def create_mlp_node_model(g, lm_config):
 
```

Comment:
Do we test the save model - load model - do inference? I think it should change a lot to our embedded layer.